README.md

```diff
@@ -0,0 +1,222 @@
+[![Coverage Status](https://coveralls.io/repos/github/digipolisantwerp/authz_module_nodejs/badge.svg?branch=master)](https://coveralls.io/github/digipolisantwerp/authz_module_nodejs?branch=master)
+# @digipolis/Authz
+
+Authorization module which can be used to check the permissions of an authenticated user.
+
+### Table of contents:
+
+<!--ts-->
+   * [Installing](#installing)
+      * [npm](#npm)
+      * [Yarn](#yarn)
+   * [Configuration](#configuration)
+   * [Installation](#installation)
+   * [Usage](#usage)
+      * [Usage as express middleware](#usage-as-express-middleware)
+      * [Usage as function](#usage-as-function)
+      * [Permissionerror](#permissionerror)
+      	  * [Model](#model)
+      	     * [Error messages](#error-messages)
+      	     * [Error detail](#error-detail)
+      	     * [Error handling](#error-handling)
+   * [Running the tests](#running-the-tests)
+   * [Versioning](#versioning)
+   * [Authors](#authors)
+   * [License](#license)
+<!--te-->
+
+
+## Installing
+
+#### npm:
+```sh
+$ npm i @digipolis/authz
+```
+
+#### Yarn:
+```sh
+$ yarn add @digipolis/authz
+```
+
+## Configuration
+
+#### Configuration for the use with the User Management Engine (UM):
+
+##### Params:
+| Param                             | Description                                                                                               | Values                                                |
+| :---                              | :---                                                                                                      | :---                                                  |
+| ***debug*** *(optional)*          | Set debugging mode                                                                                        | **true** / **false** (default)                        |
+| ***source***                      | The source to use by default. You can also specify a source in the function call                          | **authzv2** / **meauthz**                             |
+| ***sources***                     | Object with possible authz sources and their configurations                                               | `{ authzv2: { _config_ }}`                            |
+| **tokenLocation** *(optional)* | Location of the token on the request object. Used by middleware. Defaults to 'headers.authorization'      | headers.authorization / req.session.token (example)   |
```

Comment:
Misschien nog verduidelijken dat het de locatie **op het req object** is.

\+ het voorbeeld "req.session.token" moet volgens mij zonder "req" zijn.